examples/pytorch/hunyuan_image3/pipeline_hunyuan_image3_flashinfer.py

```
@@ -0,0 +1,600 @@
+        # hidden state (NOT the static KV cache) so callers hold a private copy.
+        try:
+            if getattr(out, "last_hidden_state", None) is not None:
+                out.last_hidden_state = out.last_hidden_state.clone()
+            elif torch.is_tensor(out):
+                out = out.clone()
+            elif isinstance(out, (tuple, list)) and out and torch.is_tensor(out[0]):
+                cloned = list(out)
+                cloned[0] = cloned[0].clone()
+                out = type(out)(cloned)
+        except Exception:
+            pass
```

Ruff (0.15.15)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)